### ISSUE TITLE
[HL2] The "Classic Effects" setting does not respect the Episodes

### ISSUE THREAD

**Issue by VanillaSylv** [ValveSoftware/Source-1-Games#6648]
The "classic effects" video setting sets "r_classic_fire" and "r_classic_blood" to "2" in the episodes than to "1", making those effects work only in the base game.

I don't think i need to add any steps to reproduce the issue...

**Comment by DenAkimov** [comment]
It sets them to "2" in Half-Life 2 too, but it works there. Seems intentional.

**Comment by nbmrjuhneibkr** [comment]
These effects only ever existed in the base game, so it makes sense.

**Comment by sylveonsylvia** [comment]
> These effects only ever existed in the base game, so it makes sense.
incorrect, episode 1 used the original effects, the new fire and blood effects where added in episode 2

**Comment by sylveonsylvia** [comment]
> These effects only ever existed in the base game, so it makes sense.

incorrect, episode 1 used the old effects at launch, the new fire and blood effects where added in episode 2

**Comment by nbmrjuhneibkr** [comment]
> incorrect, episode 1 used the old effects at launch, the new fire and blood effects where added in episode 2

Blood - maybe, but I'm fairly certain that E1 always had "new" fire effects since it came out.

Maybe I'm misremembering things. I don't have time to extract game data from my retail Episode One DVD to verify this right now.
